easybuild/tools/options.py

```diff
@@ -433,6 +433,16 @@ def package_options(self):
         self.log.debug("package_options: descr %s opts %s" % (descr, opts))
         self.add_group_parser(opts, descr)
 
+    def rpath_options(self):
+        # rpath Options
+        descr = ("RPATH options", "changes linking to include rpaths")
+
+        opts = OrderedDict({
+            'rpath': ("Enable RPATH support", None, 'store_true', False),
```

Comment:
do you expect other options here?

if not, this shouldn't be an option group by itself?